### PR TITLE
Rename contract interface methods

### PR DESCRIPTION
**Pull request by cf19drofxots**
Renames:
- reducer => ~~reduce~~ applyAction
- resolver => resolve

### CODE REVIEW

**Comment by IIIIllllIIIIllllIIIIllllIIIIllllIIIIll** [comment]
note that this also renames the `resolver` from the registry, which is (unfortunately) different from the resolver used in apps

**Comment by cf19drofxots** [comment]
@IIIIllllIIIIllllIIIIllllIIIIllllIIIIll good catch - reverted

File: packages/apps/contracts/CommitRevealApp.sol [hunk 1]
```diff
@@ -58,7 +58,7 @@ contract CommitRevealApp {
     return Player.CHOOSING;
   }
 
-  function reducer(AppState state, Action action)
+  function reduce(AppState state, Action action)
```
Comment (mvanderh) [inline packages/apps/contracts/CommitRevealApp.sol]: Is there a reason we don't just call this `applyAction` or something? Is it just to use Redux terminology?

Comment (IIIIllllIIIIllllIIIIllllIIIIllllIIIIll) [reply packages/apps/contracts/CommitRevealApp.sol]: I prefer `applyAction` over `reduce`

Comment (mvanderh) [reply packages/apps/contracts/CommitRevealApp.sol]: @snario @armaniferrante @ebryn

Comment (armaniferrante) [reply packages/apps/contracts/CommitRevealApp.sol]: As mentioned offline, I prefer `applyAction` over reduce as well. Reduce/reducer confused me when I first read it.

**Comment by cf19drofxots** [comment]
@mvanderh @IIIIllllIIIIllllIIIIllllIIIIllllIIIIll @armaniferrante any other concerns with this renaming?

**Pull request merged**